web/src/test/auto/dom/cases/core-adapter/basic.tests.ts

```diff
@@ -1,22 +1,22 @@
 import { assert } from 'chai';
 import { KM_Core, KM_CORE_STATUS } from 'keyman/engine/core-adapter';
 
-const coreurl = '/build/engine/obj/core-adapter/import/core';
+export const coreurl = '/build/engine/obj/core-adapter/import/core';
+
+export async function loadKeyboardBlob(uri: string) {
+  const response = await fetch(uri);
+  if (!response.ok) {
+    throw new Error(`HTTP ${response.status} ${response.statusText}`);
+  }
+
+  const buffer = await response.arrayBuffer();
+  return new Uint8Array(buffer);
+}
```

Comment:
Can we please put these into a helper module so we are not importing basic.tests.ts in other test modules?

Reply:
done